qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -371,31 +390,50 @@ def new_spin_echo_sequence(duration=None, pre_post_rotation=False, **kwargs):
 def new_carr_purcell_sequence(
     duration=None, number_of_offsets=None, pre_post_rotation=False, **kwargs
 ):
-    """
-    Creates Carr-Purcell Sequence.
+    r"""
+    Creates the Carr-Purcell sequence.
 
     Parameters
     ---------
     duration : float, optional
-        Total duration of the sequence. Defaults to None
+        Total duration of the sequence :math:`\tau`. Defaults to None.
     number_of_offsets : int, optional
-        Number of offsets. Defaults to None
+        Number of offsets :math:`n`. Defaults to None.
     pre_post_rotation : bool, optional
-        If True, a :math:`\\pi.2` rotation is added at the
+        If True, a :math:`X_{\pi/2}` rotation is added at the
         start and end of the sequence.
     kwargs : dict
-        Additional keywords required by
-        qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
+        Additional keywords required by DynamicDecouplingSequence.
 
     Returns
     -------
-    qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
-        Carr-Purcell sequence
+    DynamicDecouplingSequence
+        The Carr-Purcell sequence.
+
+    See Also
```

Comment:
I think See Also, Notes, and References might come after Raises. See https://numpydoc.readthedocs.io/en/latest/format.html